commands/version.js

```diff
@@ -14,7 +15,7 @@ class Version extends Command {
 
   static execute(args, msg) {
     const { channel } = msg;
-    let nodeEnv = process.env.NODE_ENV;
+    let nodeEnv = config.env;
```

Comment:
Assigning config.env to nodeEnv seems unnecessary. I think destructuring env off of config, then using that as the variable might be a cleaner solution.